doc/ref/configuration/master.rst

```diff
@@ -96,6 +96,23 @@ The user to run the Salt processes
 
 .. conf_master:: ret_port
 
+``enable_ssh_minions``
+----------------------
+
+
+Default: ``False``
+
+Tell the master to also use SaltSSH when running commands against minions.
```

Comment:
This has always been `salt-ssh` and not `SaltSSH`. (Though I do like that. If we change it we should change it everywhere, however.)